app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -143,15 +145,14 @@ class ProfileEditActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix ProfileEditActivityTest
-  @Ignore
   fun testProfileEditActivity_startActivityWithUserProfile_checkUserInfoIsDisplayed() {
     ActivityScenario.launch<ProfileEditActivity>(
       ProfileEditActivity.createProfileEditActivity(
         context,
         1
```

Comment:
let's use named arguments here and all places.

Reply:
DONE

Reply:
It doesn't seem to be done!! Could you please check once?

Reply:
I checked, it is done.

Reply:
<img width="613" alt="Screenshot 2020-11-05 at 08 54 26" src="https://user-images.githubusercontent.com/25057618/98194055-87b5ae00-1f44-11eb-9979-c61ac7da9e2a.png">

I didn't find the changes. Checked the latest commit.

Reply:
named arguments -> 
`profileId = 1`

Reply:
Oh sorry, I misunderstood. Will do shortly

Reply:
DONE